ClassIsland/App.xaml.cs

```diff
@@ -205,7 +205,7 @@ internal void ProcessUnhandledException(Exception e, bool critical=false)
         CrashWindow = new CrashWindow()
         {
             CrashInfo = e.ToString(),
-            AllowIgnore = _isStartedCompleted && !critical,
+            AllowIgnore = !critical,
```

Comment:
```suggestion
            AllowIgnore = _isStartedCompleted && !critical,
```

最好不要允许在启动没有完成时忽略异常，不然可能会卡死在启动画面。

Reply:
抱歉，我想的是用户有时候应用设置调错，此前可以忽略错误，修正应用设置，如常运行。但添加无法忽略判定后，只能修改应用配置文件。非常抱歉没有考虑到应用启动时用户无法通过托盘菜单退出应用的情况，感谢指正，我尽快修改。

Reply:
ok. gh 帮我改了，但没有给我改 commit 描述的地方😢若合并，请帮选压缩提交，谢谢